docs/event-bus/docs/040-details-troubleshooting.md

```diff
@@ -0,0 +1,35 @@
+
+---
+title: Troubleshooting
+type: Details
+---
+## General Troubleshooting Guidelines
+
+* If the lambda or the service does not receive any Events, check the following:
+  - Confirm that the EventActivation is in place.
+  - Ensure that the webhook defined for the lambda or the service is up and running.
+  - Make sure the Events are published.
+
+* If errors appear while sending Events:
+  - Check if the `publish` application is up and running.
+  - Make sure that NATS Streaming is up and running.
+
```

Comment:
Maybe a kind of closing sentence would be helpful? Something like:
`If these general guidelines do not help, go to the next section of this document.` 
What is your opinion????